fendermint/vm/interpreter/Cargo.toml

```diff
@@ -43,6 +45,7 @@ fvm_shared = { workspace = true }
 fvm_ipld_blockstore = { workspace = true }
 fvm_ipld_encoding = { workspace = true }
 fvm_ipld_car = { workspace = true }
+fil_actors_runtime = { workspace = true }
```

Comment:
Yeah so this shouldn't be needed as we are not running an actor inside the interpreter.